src/transformers/modeling_flax_utils.py

```diff
@@ -70,6 +70,13 @@ def quick_gelu(x):
 }
 
 
+def mask_value(dtype):
```

Comment:
IMO it's not worth a funtion for a single line. That "over-abstracted" would prefer to have this line always in the modeling forward pass